File: busca-ordenacao/javascript/time-test.js
function myFunction() {
    let x = 10;
    for (let i = 0; i < 100000000; i++) {
        x = -x;
    }
}

let start = Date.now();
myFunction();
let end = Date.now();

let time = end - start;
console.log(`Duration: ${time}ms`);

File: busca-ordenacao/java/src/TimeTest.java
import java.time.Duration;
import java.time.Instant;

public class TimeTest {

	static void myFunction() {
		double x = 10;
		for (long i = 0; i < 100000000L; i++) {
			x = -x;
		}
	}
	
	public static void main(String[] args) {
		Instant start = Instant.now();
		myFunction();
		Instant end = Instant.now();
		long time = Duration.between(start, end).toMillis();
		System.out.printf("Duration: %dms\n", time);
	}
}
